homeassistant/components/otbr/config_flow.py

```diff
@@ -53,6 +58,23 @@ async def async_step_hassio(self, discovery_info: HassioServiceInfo) -> FlowResu
 
         config = discovery_info.config
         url = f"http://{config['host']}:{config['port']}"
+
+        api = python_otbr_api.OTBR(url, async_get_clientsession(self.hass), 10)
+        try:
+            if await api.get_active_dataset_tlvs() is None:
```

Comment:
Interesting doing it here. I thought that we would do it during setup, so we also cover manual setups?